pyproject.toml

```
@@ -1,7 +1,10 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "setuptools-scm", "cython"]
+requires = ["setuptools >= 40.6.0", "setuptools-scm >= 4", "cython"]
```

In order to have setuptools-scm read from `pyproject.toml` we need to specify

```suggestion
requires = ["setuptools >= 40.6.0", "setuptools_scm[toml] >= 4", "cython"]
```

https://github.com/pypa/setuptools_scm#pyprojecttoml-usage